app.py

```diff
@@ -38,13 +39,30 @@
 
 """CREATE DATABASE ACCESS OBJECT AND CACHE SEARCH OPTIONS"""
 augur = AugurManager()
+
+# make sure that parameters for Augur connection have been supplied.
+client_secret = os.getenv("AUGUR_CLIENT_SECRET", "")
+app_id = os.getenv("AUGUR_APP_ID", "")
+session_endpoint = os.getenv("AUGUR_SESSION_GENERATE_ENDPOINT", "")
+groups_endpoint = os.getenv("AUGUR_USER_GROUPS_ENDPOINT", "")
+
+if not all([client_secret, app_id, session_endpoint, groups_endpoint]):
+    logging.critical("ERROR: Client Augur credentials incomplete; can't start.")
```

Comment:
A little confused here, shouldnt the app still start even without those credentials?

Reply:
yeah, need to make it so that either the login is enabled or all of those are set.

Reply:
"ERROR: Client Augur credentials incomplete; can't start." shouldnt this still be able to start without the log in necessary variables?

Reply:
If you have AUGUR_LOGIN_ENABLED=True then you need to credentials, otherwise it'll skip this step.